src/Config.php

```diff
@@ -25,10 +21,9 @@
  *
  * @psalm-import-type ServiceManagerConfigurationType from ConfigInterface
  */
-class Config implements ConfigInterface
+final class Config implements ConfigInterface
 {
-    /** @var array<string,bool> */
-    private array $allowedKeys = [
+    private const ALLOWED_CONFIGURATION_KEYS = [
```

Comment:
This constant is now unused

Reply:
Dropped the whole file, we are targeting next major anyways and thus can be dropped.
Have to mark it as deprecated in v3 after merging this PR tho.